website/pages/deployment/stackinfo.mdx

```diff
@@ -0,0 +1,112 @@
+import {Callout} from "nextra/components"
+
+# Stack Components
+
+The All In One stack provides a number of services which would
+usually be hosted separately, but are combined together here
+for the convenience of users.
+
+Below these are listed, along with a short description of their function,
+and with the local web addresses and user accounts.
+Note that these user accounts are only intended for local training purposes.
+If you deploy this stack for public access then these must be changed.
+
+## TRE-FX Website (Submission Layer)
+
+<Callout>
+**Web Address:** [http://localhost:8888](http://localhost:8888)
+</Callout>
+
+This is the front page of the stack, which any researchers interacting
+with the service will use.
+
+<Callout>
+**Administration account**
+- Username: `globaladminuser`
+- Password: `password123`
+</Callout>
+
+## KeyCloak Server
+
+<Callout>
+**Web Address:** [htpp://localhost:8085](http://localhost:8085)
+</Callout>
+
+The user management service for the stack.
+
+<Callout>
+**Administration account**
+- Username: `admin`
+- Password: `admin`
+</Callout>
+
+## TRE Management Website (TRE Layer)
+
+<Callout>
+**Web Address:** [htpp://localhost:8989](http://localhost:8989)
+</Callout>
+
+TRE management services.
+
+<Callout>
+**Administration account**
+- Username: `globaladminuser`
+- Password: `password123`
+</Callout>
+
+## Egress Layer
+
+<Callout>
+**Web Address:** [htpp://localhost:8100](http://localhost:8100)
+</Callout>
+
+Egress management services.
+
+<Callout>
+**Administration account**
+- Username: `globaladminuser`
+- Password: `password123`
+</Callout>
+
+## Backend Services
+
+### Seq Activity Logs
+
+<Callout>
+**Web Address:** [htpp://localhost:5341](http://localhost:5341)
+</Callout>
+
+Activity logs for the stack.
+
+### TRE API
+
+<Callout>
+**Web Address:** [htpp://localhost:8072/swagger](http://localhost:8072/swagger)
+</Callout>
+
+SWAGGER documentation for the TRE service.
+
+### TRE Job Tracking
+
+<Callout>
+**Web Address:** [htpp://localhost:8072/swagger](http://localhost:8072/hangfire)
+</Callout>
+
+Job tracking service for the TRE service.
+
+<Callout>
+**Administration account**
+- Username: `admin`
+- Password: `password123`
+</Callout>
+
+### Min IO Services
+
+<Callout>
+- **MinioAAA Web Address:** [htpp://localhost:9000](http://localhost:9000)
+- **Minio2 Web Address:** [htpp://localhost:9000](http://localhost:9002)
```

Comment:
There's also the demo login credentials for the MinIO those should they be added here? its minio and minio123

Reply:
that's helpful thanks - I think we should add them, I just couldn't remember what they were.

Reply:
I can't get these logins to work - could you put the details for each of the servers here for me?

Reply:
ignore me - I've seen from your PR how to use these, I'll add them in now.